doc/2/guides/advanced/streams/index.md

```diff
@@ -0,0 +1,57 @@
+---
+code: false
+type: page
+title: Stream large volumes
+description: Create streams to send large amount of data over the network
+order: 300
+---
+
+# HTTP Streams <SinceBadge version="auto-version" />
```

Comment:
This section should rather be in the **Develop On Kuzzle** > **API Controller** section (We have a [Response Format](https://docs.kuzzle.io/core/2/guides/develop-on-kuzzle/api-controllers/#response-format) section)

Reply:
I think this section might be useful here too if someone stumble uppon this page, what do you think ?
should we duplicate this section or juste move it ?

Reply:
IMHO it's not a very common usage to manipulate HTTP streams.  Streams are more a subset of what you can do when defining API action than an entire feature on it's own.  
About duplication, it's generally not a good idea I think